lib_eio/flow.mli

```diff
@@ -146,5 +151,8 @@ module Pi : sig
     | Source : ('t, (module SOURCE with type t = 't), [> source_ty]) Resource.pi
     | Sink : ('t, (module SINK with type t = 't), [> sink_ty]) Resource.pi
     | Shutdown : ('t, (module SHUTDOWN with type t = 't), [> shutdown_ty]) Resource.pi
+
+  val simple_copy : single_write:('t -> Cstruct.t list -> int) -> 't -> src:_ source -> unit
```

Comment:
Would a better name for this function be "serial_copy"?  "simple" is subjective, but "serial" describes what it does.